pkg/operator/controllerimagepruner.go

```diff
@@ -0,0 +1,384 @@
+package operator
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+	"time"
+
+	regopclient "github.com/openshift/cluster-image-registry-operator/pkg/client"
```

Comment:
Usually we use this order of packages:

1. standard packages
2. other packages
3. k8s
4. openshift
5. the current repository

Reply:
I know that.  When the operator was refactored to use the openshift/api everything got in the wrong order and I didn't feel like fixing it all, but I will fix this one.